src/components/landingPage/listOfArticles/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ArticleCard from '../../common/articleCard';
+import * as actionTypes from '../../../redux/actions/actionTypes';
+
+export default function ListOfArticles({ articles, limitArticlesNumber }) {
```

Comment:
You should move this component to common. I want anyone implementing list of articles to reuse it.